classes/fields/avatar.php

```diff
@@ -144,7 +145,7 @@ public function get_avatar( $avatar, $id_or_email, $size, $default = '', $alt =
 							$attributes['alt'] = $alt;
 						}
 
-						$user_avatar = pods_image( $user_avatar, array( $size, $size ), 0, $attributes );
+						$user_avatar = pods_image( $user_avatar, array( $size, $size ), 0, $attributes, true );
```

Comment:
```suggestion
						$user_avatar = pods_image( $user_avatar, array( $size, $size, 1 ), 0, $attributes, true );
```

We might need to make the image crop so that it enforces the perfect square.

Reply:
This is the default value but good with me!